ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecureRequest.java

```diff
@@ -56,6 +58,7 @@ public class ThreeDSecureRequest implements Parcelable {
 
     private String nonce;
     private String amount;
+    private Map<String,Object> customFields;
```

Comment:
```suggestion
    private Map<String, String> customFields;
```
Can we change this to align with iOS? `Object` in Java is like `Any` in Swift. Changing to Map<String, String> would align with iOS's [[String: String]](https://github.com/braintree/braintree_ios/blob/f1444f07489bbb306a3adf2e18c6c493036ff55a/Sources/BraintreeThreeDSecure/BTThreeDSecureRequest.swift#L16) interface better.